civitai-downloader-v2/src/cli/main.py

```
@@ -1080,12 +1107,188 @@ async def download_with_semaphore(model_id):
+        try:
+            cli_context = CLIContext()
+            await cli_context.initialize()
+            
+            if verbose:
+                logging.getLogger().setLevel(logging.DEBUG)
+                click.echo(f"Saving models from {jsonl_path} to database...")
+            
+            # Save models to database
+            saved_count, skipped_count = cli_context.model_storage.save_models_from_jsonl(jsonl_path)
+            
+            click.echo(f"✅ Database save completed:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)